app/src/User/AuthApi.php

```diff
@@ -29,9 +29,9 @@ public function login($username, $password, $clientId, $clientSecret)
         if ($result) {
             $data = json_decode($result);
             if ($data) {
-                if (isset($data->access_token)) {
+//                if (isset($data->access_token)) {
```

Comment:
I'd remove it completely. The history contains it anyway and when it's not needed you can just get rid of it.